actors/builtin/miner/miner_actor.go

```diff
@@ -541,8 +539,9 @@ func (a *Actor) OnDeferredCronEvent(rt Runtime, params *OnDeferredCronEventParam
 	}
 
 	if payload.EventType == CronEventType_Miner_TempFault {
+		// should roll the huge number of state transactions in this loop into ~1
```

Comment:
```suggestion
		// TODO: should roll the huge number of state transactions in this loop into ~1
```

Reply:
(makes this easier to grep for)

Reply:
I've been reserving TODO temporarily for things that we absoluty must do, but this one is fixed by #135 anyway.